src/v3/src/transformer/webauthn/transformWebAuthNAuthenticator.test.ts

```diff
@@ -86,29 +91,48 @@ describe('WebAuthN Transformer Tests', () => {
       const updatedFormBag = transformWebAuthNAuthenticator({ transaction, formBag, widgetProps });
 
       // Verify added elements
-      expect(updatedFormBag.uischema.elements.length).toBe(3);
-      expect(updatedFormBag.uischema.elements[0].type).toBe('Title');
-      expect((updatedFormBag.uischema.elements[0] as TitleElement).options.content)
-        .toBe('oie.enroll.webauthn.title');
-      expect(updatedFormBag.uischema.elements[1].type).toBe('Description');
-      expect((updatedFormBag.uischema.elements[1] as DescriptionElement).options.content)
-        .toBe('oie.enroll.webauthn.instructions');
-      expect(updatedFormBag.uischema.elements[2]?.type).toBe('WebAuthNSubmitButton');
-      expect((updatedFormBag.uischema.elements[2] as WebAuthNButtonElement).options?.onClick)
-        .not.toBeUndefined();
-      expect((updatedFormBag.uischema.elements[2] as WebAuthNButtonElement).options?.label)
-        .toBe('oie.enroll.webauthn.save');
-      expect((updatedFormBag.uischema.elements[2] as WebAuthNButtonElement).options?.submitOnLoad)
-        .toBe(true);
-      expect((updatedFormBag.uischema.elements[2] as WebAuthNButtonElement)
-        .options?.showLoadingIndicator).toBe(true);
+      expect(updatedFormBag.uischema.elements.length).toBe(4);
+      expect(updatedFormBag).toMatchSnapshot();
+    });
+
+    it('should render title, description, button, and callout elements when WebAuthN API is available', () => {
+      transaction.nextStep!.relatesTo = {
+        value: {
+          contextualData: {
+            activationData: {
+              authenticatorSelection: { userVerification: 'required' },
+            } as unknown as ActivationData,
+          },
+        } as unknown as IdxAuthenticator,
+      };
+      mockIsEdgeBrowser = true;
```

Comment:
If relevant, maybe "Edge" should appear somewhere in the test title? e.g., `it("should ... on MS Edge"). Same with Safari